docs/how-to-use.md

```diff
@@ -12,11 +12,11 @@ Commands are accessible through :
 - **New File** : opens an unsaved file with links to help you and a code snippet that you can save as `code.py` / `main.py`.  
   _(**Note :** will open the simulator webview if it's not open yet)_.
 
-- **Run Simulator** : run the code you have open on the simulator (make sure you've clicked on a valid code file).  
+- **Run Simulator** : runs the code you have opened on the simulator (make sure you've clicked on a valid code file).  
```

Comment:
I'm not a native speaker so I was wondering how does "opened **in** the simulator" sound compared to "opened **on** the simulator"?

Reply:
I agree with you that it sounds better. Will update. :)